scss/_carousel.scss

```diff
@@ -302,8 +302,41 @@
 }
 
 /* rtl:end:ignore */
-// End mod
 
+.carousel-indicators-wrapper {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  z-index: 2;
+  display: flex;
+  transform: translateX(-50%);
+
+  .carousel-control-play-pause {
```

Comment:
Is there no specific rendering for the hover state?

Reply:
@MewenLeHo You've added a hover state, but is it confirmed and validated by the designers?

Reply:
Yes, validated by @Franco-Riccitelli during our meeting yesterday.